spec/requests/api/moves_controller_update_v2_spec.rb

```diff
@@ -40,7 +40,6 @@
           additional_information: 'some more info',
           cancellation_reason: nil,
           cancellation_reason_comment: nil,
-          move_type: 'court_appearance',
```

Comment:
The API endpoint doesn't support changing this attribute - it's not in the included list of permitted params so not really sure why it's present here.